drivers/docker/coordinator.go

```diff
@@ -9,13 +9,13 @@ import (
 	"fmt"
 	"io"
 	"regexp"
-	"strings"
 	"sync"
 	"time"
 
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/api/types/image"
 	"github.com/docker/docker/api/types/registry"
+	"github.com/docker/docker/errdefs"
```

Comment:
This `errdefs` package is so weird if you go read the code. Instead of using the stdlib `errors.Is`, it does a bunch of type assertions. I guess this is ok if it's errors that they emit, so long as they never wrap them before emitting them?

Reply:
I checked with dlv, they don't. But yes it's weird and it's bad. Docker is bad.